src/plopp/backends/plotly/canvas.py

```diff
@@ -58,15 +58,9 @@ def __init__(
         # Instead, we forward all the kwargs from the figure to both the canvas and the
         # artist, and filter out the artist kwargs with `**ignored`.
 
-        ymin, ymax, logy = parse_vmin_vmax_norm(
-            vmin=user_vmin,
-            vmax=user_vmax,
-            norm=norm,
-            ymin=ymin,
-            ymax=ymax,
-            log=logy,
-            y_or_c='y',
-        )
+        ymin = parse_mutually_exclusive(vmin=user_vmin, ymin=ymin)
+        ymax = parse_mutually_exclusive(vmax=user_vmax, ymax=ymax)
+        logy = parse_mutually_exclusive(norm=norm, log=logy)
```

Comment:
I don't think this works because `logy` needs to be a bool. But `norm` would be `'log'`.